libs/GraphicECS/SFML/Systems/SoundManagement.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundManagement
+*/
+
+#include "SoundManagement.hpp"
+
+using namespace graphicECS::SFML::Components;
+using namespace graphicECS::SFML::Resources;
+
+namespace graphicECS::SFML::Systems
+{
+    void SoundManagement::run(World &world)
+    {
+        std::vector<std::shared_ptr<ecs::Entity>> joined = world.joinEntities<SoundComponent>();
+
+        auto stopSound = [&world](std::shared_ptr<ecs::Entity> entityPtr) {
+            auto guard = std::lock_guard(*entityPtr.get());
+            SoundComponent soundComponent = entityPtr.get()->getComponent<SoundComponent>();
+            sf::Sound sound;
+
+            sound.setBuffer(*world.getResource<SoundResource>()._soundsList.at(soundComponent.sound_e));
+
+            if (soundComponent._status == SoundComponent::STOPED && sound.getStatus() != sf::Sound::Stopped) {
+                sound.stop();
+            }
+            if (soundComponent._status == SoundComponent::PAUSED && sound.getStatus() != sf::Sound::Paused) {
+                sound.pause();
```

Comment:
lock this one

Reply:
Don't need to lock this one because i don't stock this variable and sf::sound is not a Mutex